interfaces/builtin/dbus_test.go

```diff
@@ -256,6 +299,200 @@ slots:
 	c.Assert(interfaces.BeforePrepareSlot(s.iface, slot), IsNil)
 }
 
+func (s *DbusInterfaceSuite) TestSanitizeSlotActivatableOk(c *C) {
+	var mockSnapYaml = `name: dbus-snap
+version: 1.0
+slots:
+  dbus-service-slot:
+    interface: dbus
+    bus: session
+    name: org.dbus-snap.session
+apps:
+  app-dbus-slot:
+    daemon: simple
+    daemon-scope: user
```

Comment:
Super late design nitpick time: could we use `daemon-scope: user` to derive that a snap uses the session bus? Is there a valid reason for a user session app to grab a name on the system bus?